profile/README.md

```diff
@@ -1,49 +1,87 @@
-### Welcome to TrustyAI 👋
-TrustyAI is an open source AI Explainability (XAI) Toolkit. 
-It provides a Java and Python library and [Open Data Hub](https://github.com/opendatahub-io) (ODH) component for XAI explanations of predictive models for both enterprise and data science. 
+# Welcome to TrustyAI 👋
 
-The [project roadmap](https://github.com/orgs/trustyai-explainability/projects/10) offers a view on new tools and integration the project developers are planning to add.
-
-Included in the TrustyAI core library are several tools, which include:
-
-- Local explainers
-- Global explainers
-- Time-series explainers
+[TrustyAI](https://trustyai-explainability.github.io/trustyai-site/main/main.html) is an open source Responsible AI toolkit supported by Red Hat and IBM. TrustyAI provides tools for a variety of responsible AI workflows, such as:
+- Local and global model explanations
 - Fairness metrics
 - Drift metrics
-- Language performance metrics
-- Generative text model evaluations and benchmarks 
+- Text detoxification
+- Language model benchmarking
+- Language model guardrails
 
-TrustyAI also provides the following integrations:
+TrustyAI is a default component of [Open Data Hub](https://opendatahub.io/) and [Red Hat Openshift AI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai), and has integrations with projects like [KServe](https://www.google.com/search?q=kserve&sourceid=chrome&ie=UTF-8), [Caikit](https://www.google.com/search?q=caikit&sourceid=chrome&ie=UTF-8), and [vLLM](https://github.com/vllm-project/vllm).
```

Comment:
Would it be better to have direct links to the homepages, rather than a google search?

Reply:
whoops